src/uct/api/uct.h

```diff
@@ -699,15 +699,21 @@ enum uct_md_mem_flags {
     UCT_MD_MEM_ACCESS_REMOTE_PUT    = UCS_BIT(5), /**< enable remote put access */
     UCT_MD_MEM_ACCESS_REMOTE_GET    = UCS_BIT(6), /**< enable remote get access */
     UCT_MD_MEM_ACCESS_REMOTE_ATOMIC = UCS_BIT(7), /**< enable remote atomic access */
+    UCT_MD_MEM_ACCESS_LOCAL_WRITE   = UCS_BIT(8), /**< enable local write access */
+    UCT_MD_MEM_ACCESS_LOCAL_READ    = UCS_BIT(9), /**< enable local read access */
```

Comment:
@leibin2014 can you pls switch the flags order (read <-> write)?

Reply:
@yosefe I made this order refered to the order of _REMOTE_PUT/_REMOTE_GET. But for sure, I will switch the order of _LOCAL_READ/WRITE.

Reply:
@yosefe Done